### PR TITLE
Announce community review of Eclipse projects

### PR DESCRIPTION
**Pull request by aahlenst**

### CODE REVIEW

File: content/blog/eclipse-adoptium-projects-ready-for-community-review/index.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+* **Eclipse Temurin** (an anagram of "runtime") focuses on building and releasing JDKs (that part that most associate with AdoptOpenJDK).
+
+AdoptOpenJDK's application to become an Eclipse project was approved under the provision that we only release binaries that have passed the respective TCK. The [TCK for Java SE](https://openjdk.java.net/groups/conformance/JckAccess/) is a proprietary technology of Oracle and to be kept secret -- something that is in stark contrast to the spirit of open source. The Eclipse Foundation and AdoptOpenJDK went to great lengths to reconcile Oracle's wishes with the realities of open source. The result is the [Eclipse Temurin Compliance](https://projects.eclipse.org/proposals/eclipse-temurin-compliance) project, which governs access to TCK material and resources. Thanks to the Eclipse Temurin Compliance project, most contributors can continue their work without being impacted by the TCK requirement. We thank Eclipse Foundation's Executive Director Mike Milinkovich and his team for reaching an agreement with Oracle and making this possible.
```
Comment (karianna) [inline content/blog/eclipse-adoptium-projects-ready-for-community-review/index.md:17]: Probably need to remove `something that is in stark contrast to the spirit of open source`

Comment (karianna) [reply content/blog/eclipse-adoptium-projects-ready-for-community-review/index.md:17]: And change `The Eclipse Foundation and AdoptOpenJDK went to great lengths to reconcile Oracle's wishes with the realities of open source` to `The Eclipse Foundation and AdoptOpenJDK worked at length with Oracle to arrange access` or something like that.

**Review by karianna — CHANGES_REQUESTED** [review]
A couple of suggested changes.

Also, it's worth addressing the other AdoptOpenjDK projects like JitWatch.  I think we could add something like "We will be working with each of the AdoptOpenJDK projects not directly related to the production of runtimes to see whether they would like to also move across as a project under the Ecliose Adoptium working group or whether they would prefer to stay under the AdoptOpenJDK org on GitHub.

File: content/blog/eclipse-adoptium-projects-ready-for-community-review/index.md (hunk 1 above)
Comment (jerboaa) [reply content/blog/eclipse-adoptium-projects-ready-for-community-review/index.md:17]: I concur. We should find more neutral words.

Comment (tellison) [reply content/blog/eclipse-adoptium-projects-ready-for-community-review/index.md:17]: +1 to simply deleting the portion Martijn highlights.  Your explanation stands on its own without it.  All owners have the right **not** to make their code available under an open source license.

Comment (smlambert) [reply content/blog/eclipse-adoptium-projects-ready-for-community-review/index.md:17]: The words are true and accurate.  If you wish to remove the emotion and human aspect from them, then perhaps...  

"and to be kept secret -- while antithetical to the spirit of open source, nonetheless a requirement.  The Eclipse Foundation and AdoptOpenJDK took time and undertook the necessary discourse to reconcile Oracle's requirement with the realities of open source."

File: content/blog/eclipse-adoptium-projects-ready-for-community-review/index.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+tags:
+  - AdoptOpenJDK
+  - Eclipse
+  - Foundation
+---
+
+Six months ago, we announced that AdoptOpenJDK is going to join the Eclipse Foundation and rebrand as Eclipse Adoptium. After a long period of work behind the scenes, the first batch of project proposals is ready for community review. At the Eclipse Foundation, the monolithic AdoptOpenJDK project will be split into multiple sub-projects under the umbrella of the [Eclipse Adoptium Top Level Project](https://projects.eclipse.org/projects/adoptium):
+
+* **Eclipse AQAvit** encompasses all our activities around testing, especially the development of our [AQA test suite](/2019/07/the-first-drop-introducing-adoptopenjdk-quality-assurance-aqa-v1-0/).
+* **Eclipse Temurin** (an anagram of "runtime") focuses on building and releasing JDKs (that part that most associate with AdoptOpenJDK).
```
Comment (smlambert) [inline content/blog/eclipse-adoptium-projects-ready-for-community-review/index.md:15]: Do not expect you to change the text, just wanted to include in the review for posterity...

Temurin is not only an anagram of "runtime" but a chemical compound, 1,3,7,9-tetramethyluric acid similar to caffeine... (with the bonus of being demonstratively non-habituating and clinically safe)
Screen Shot 2020-12-21 at 12 15 48 PM

**Review by karianna — APPROVED** [review]

**Review by jerboaa — APPROVED** [review]
This looks good. Nice work!

**Pull request merged**